comictaggerlib/__init__.py

```diff
@@ -1 +1,5 @@
 from __future__ import annotations
+
+import importlib.resources
+
+data_path = importlib.resources.files(__package__)
```

Comment:
Not sure why this is here... any resources should be their own package

Reply:
Give it it's own package like `publishers.json` is in it's own comicapi.data package

Reply:
I've made a dir called `plugin_manifest` in `comictaggerlib`